configuration/concepts/common.csv

```diff
@@ -0,0 +1,74 @@
+144dd787-a13f-437d-b6c7-9a7d4cf3bc56,,CIEL:162398,Dose,Dose,Dose,Dose,,,Misc,N/A,,,,,,,,,,,
+f36097e7-7d66-4196-8d95-6b66f31435cc,,,See comments,Commentaires,See comments,Commentaires,,,Frequency,N/A,,,,,,,,,,,
+9d5b708e-3f10-11e4-adec-0800271c1b75,,CIEL:162384,Dosing Units,Unités de dosage,dosing units,Unités de dosage,,,Misc,N/A,,,Capsule(s); Tablet(s); ml; mg; IU; Drop; Tablespoon; Teaspoon; Unit(s); Puff(s); Application; Packet; Dose; See comments,,,,,,,,
+0de96a2d-5c2c-4110-9770-914369de1713,,,Opthalmic,Ophtalmique,Opthalmic,Ophtalmique,,,Misc,N/A,,,,,,,,,,,
+99cbeb1b-621c-4645-a600-d702364d9885,,,Otic,Otique,Otic,Otique,,,Misc,N/A,,,,,,,,,,,
+9d6c9153-3f10-11e4-adec-0800271c1b75,,CIEL:162392,Intravaginal,Intravaginale,Intravaginal,Intravaginale,,,Misc,N/A,,,,,,,,,,,
+9d697972-3f10-11e4-adec-0800271c1b75,,CIEL:162394,Drug Routes,Voie d'administration,Drug Routes,Voie d'administration,,,Misc,N/A,,,Oral; Topical; Inhalation; Intramuscular; Intravenous; Intravaginal; Opthalmic; Otic; See comments;,,,,,,,,
+9d77abd7-3f10-11e4-adec-0800271c1b75,,CIEL:160862,Daily,du quotidien,OD,OD,,,Frequency,N/A,,,,,,,,,,,
+334ce904-8a92-11e4-977f-0800271c1b75,,,Q4H,Q4H,Q4H,Q4H,,,Frequency,N/A,,,,,,,,,,,
+334dbd41-8a92-11e4-977f-0800271c1b75,,CIEL:160870,Q6H,Q6H,Q6H,Q6H,,,Frequency,N/A,,,,,,,,,,,
+334ea16c-8a92-11e4-977f-0800271c1b75,,CIEL:160866,Q8H,Q8H,Q8H,Q8H,,,Frequency,N/A,,,,,,,,,,,
+334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,Q12H,Q12H,Q12H,,,Frequency,N/A,,,,,,,,,,,
+335071ea-8a92-11e4-977f-0800271c1b75,,,Q48H,Q48H,Q48H,Q48H,,,Frequency,N/A,,,,,,,,,,,
+33516e22-8a92-11e4-977f-0800271c1b75,,CIEL:1099,Weekly,Hebdomadaire,Weekly,Hebdomadaire,,,Frequency,N/A,,,,,,,,,,,
+9d8e6918-3f10-11e4-adec-0800271c1b75,,CIEL:161235,Now,Maintenant,STAT,STAT,,,Frequency,N/A,,,,,,,,,,,
+2a148983-c143-4649-b4be-f8845c934ec0,,,Nightly,Chaque nuit,Nightly,Chaque nuit,,,Frequency,N/A,,,,,,,,,,,
+,,CIEL:1065,Yes,Oui,Yes,Oui,,,Misc,N/A,,,,,,,,,,,
+,,CIEL:1066,No,Non,No,Non,,,Misc,N/A,,,,,,,,,,,
+26c0366b-83e4-4401-9699-e545219b8870,,CIEL:5622,Other,Autre,Other,Autre,,,Misc,N/A,,,,,,,,,,,
+75e366a1-c441-4c94-9c53-312c9b8cc550,,CIEL:1175,Not Applicable,Non applicable,N/A,N/A,,,Misc,N/A,,,,,,,,,,,
+6b32bbbe-8a7c-4160-99ff-0dd6b255f458,,CIEL:1067,Unknown,Inconnu,Unknown,Inconnu,,,Misc,N/A,,,,,,,,,,,
+036f30c9-ccff-4752-8206-722724cbbf49,,CIEL:1115,Normal,Ordinaire,Normal,Ordinaire,,,Misc,N/A,,,,,,,,,,,
+7c4b5dff-ade6-4fab-9452-b3be4bcba89e,,CIEL:1116,Abnormal,Anormal,Abnormal,Anormal,,,Misc,N/A,,,,,,,,,,,
+f7139b53-6e10-4096-8e87-ea133fc8fc46,,CIEL:164371,Not Seen,Non vu,Not Seen,Non vu,,,Misc,N/A,,,,,,,,,,,
+48be742e-e22e-4bd9-bb2b-0d4840c60aeb,,,Boy,Garçon,Boy,Garçon,,,Misc,N/A,,,,,,,,,,,
+7adb833e-9e48-4b76-aa8f-24daa3acb810,,,Girl,Fille,Girl,Fille,,,Misc,N/A,,,,,,,,,,,
+5c75e55f-845c-4df6-a6da-b45475a4a215,,,Weight Percentile,Percentile,Weight Percentile,Percentile,,,Misc,Numeric,,,,,,,,,,,
```

Comment:
I am surprised there isn't a CIEL mapping for such a generic concept.

Reply:
An oversight on my end..Added